backend/module/eCampApi/src/eCampApi/V1/Rpc/Profile/ProfileController.php

```diff
@@ -67,10 +67,15 @@ private function getAction(User $user) {
             'mail' => $user->getTrustedMailAddress(),
             'role' => $user->getRole(),
             'language' => $user->getLanguage(),
-            'birthday' => Util::extractDate($user->getBirthday()),
+            'birthday' => null !== $user->getBirthday() ? $user->getBirthday() : null,
```

Comment:
This ternary operator is useless
```suggestion
            'birthday' => $user->getBirthday(),
```